ara/cli/host.py

```diff
@@ -48,7 +67,49 @@ def get_parser(self, prog_name):
         return parser
 
     def take_action(self, parsed_args):
-        host = (models.Host.query
-                .filter((models.Host.id == parsed_args.host) |
-                        (models.Host.name == parsed_args.host)).one())
-        return utils.fields_from_object(FIELDS, host)
+        hosts = (models.Host.query
+                 .filter((models.Host.id == parsed_args.host) |
+                         (models.Host.name == parsed_args.host)).one())
+
+        return utils.fields_from_object(FIELDS, hosts)
+
+
+class HostDelete(Command):
```

Comment:
What's the intended purpose ? Just deleting a host from the hosts table ? What about the other tables like stats or task results ? Could this lead to an inconsistent state where a playbook run is no longer "true" because one of the hosts involved in it was deleted ?